pymongoexplain/commands.py

```diff
@@ -77,24 +107,33 @@ def command_name(self):
 class AggregateCommand(BaseCommand):
     def __init__(self, collection: Collection, pipeline, session,
                  cursor_options,
-                 kwargs, exclude_keys = []):
-        super().__init__(collection.name)
-        self.command_document = {"pipeline": pipeline, "cursor": cursor_options}
+                 kwargs):
+
+        super().__init__(collection.name, kwargs.get("collation", None))
```

Comment:
We should be using `kwargs.pop("collation", None)` instead.

Reply:
Done.